llvm/include/llvm/CodeGenTypes/LowLevelType.h

```diff
@@ -134,15 +134,17 @@ class LLT {
 
   explicit LLT(MVT VT);
 
-  constexpr bool isValid() const { return IsScalar || RawData != 0; }
+  constexpr bool isValid() const { return IsScalar || IsPointer || IsVector; }
```

Comment:
Writing `isValid` this way just makes it clear that there is no need to call it from the other `is` methods below, since they are all testing at least one of the `Is` flags already.